src/Domain/Reviewer/Exception/ReviewerException.php

```diff
@@ -10,8 +10,6 @@
 
 namespace DemoCQRSHooksUsage\Domain\Reviewer\Exception;
 
-use PrestaShop\PrestaShop\Core\Domain\Exception\DomainException;
-
-class ReviewerException extends DomainException
+class ReviewerException extends \Exception
```

Comment:
In order to be closer to a real-world module usecase, I do not extend DomainException